webapp/lib/model.g.dart

```diff
@@ -417,12 +417,43 @@ void listenForUpdates<T>(
   });
 }
 
+/// A snapshot of a document's id and data at a particular moment in time.
 class DocSnapshot {
   final String id;
   final Map<String, dynamic> data;
 
   DocSnapshot(this.id, this.data);
 }
 
+/// A batch update, used to perform multiple writes as a single atomic unit.
+/// None of the writes are committed (or visible locally) until
+/// [DocUpdate.commit()] is called.
+abstract class DocUpdate {
+  /// Commits all of the writes in this write batch as a single atomic unit.
```

Comment:
They're not atomic - that would only be the case if we were using transactions - could you adjust the comments here  to 'batch'?

Reply:
@lukechurch  According to the [WriteBatch online docs](https://firebase.google.com/docs/reference/android/com/google/firebase/firestore/WriteBatch), it is atomic.

Please let me know how you would like to proceed and I can update the comment in a future PR.